tests/containers/podman_quadlet.pm

```diff
@@ -67,30 +102,52 @@ _EOF_
     # start the generator and check whether the files are generated
     systemctl("daemon-reload");
     check_unit_states();
-    systemctl("is-active sleeper.service", expect_false => 1);
-    systemctl("is-active sleeper-volume.service", expect_false => 1);
+    for my $unit (@units) {
+        systemctl("is-active $unit", expect_false => 1);
```

Comment:
Is the intention here to check if the units are inactive or to check if they exist? Because this will also pass if the unit doesn't exist.

Reply:
This is to check if the unit is inactive. I used the solution that was there before my improvements but I'm open to better solutions...

Reply:
If that's the purpose, then this works as expected. Perhaps it would be also good to check if the unit is present. We could use a simpe `systemctl status` command for it.

Reply:
I believe that's handled by the `check_unit_states` subroutine, validating that the unit has been "generated" if no other argument is supplied.